stdlib/3/sys.pyi

```diff
@@ -5,120 +5,132 @@
 
 from typing import (
     List, NoReturn, Sequence, Any, Dict, Tuple, TextIO, overload, Optional,
-    Union, TypeVar, Callable, Type,
+    Union, TypeVar, Callable, Type
 )
 import sys
 from types import FrameType, TracebackType
 
+if sys.version_info >= (3, 3):
+    from importlib.abc import MetaPathFinder
+else:
+    MetaPathFinder = Any
+
 _T = TypeVar('_T')
 
 # ----- sys variables -----
-abiflags = ...  # type: str
-argv = ...  # type: List[str]
+abiflags: str
+argv: List[str]
 base_exec_prefix: str
 base_prefix: str
-byteorder = ...  # type: str
-builtin_module_names = ...  # type: Sequence[str] # actually a tuple of strings
-copyright = ...  # type: str
+byteorder: str
+builtin_module_names: Sequence[str]  # actually a tuple of strings
+copyright: str
 # dllhandle = 0  # Windows only
 dont_write_bytecode: bool
-__displayhook__ = ...  # type: Any # contains the original value of displayhook
-__excepthook__ = ...  # type: Any  # contains the original value of excepthook
-exec_prefix = ...  # type: str
-executable = ...  # type: str
-float_repr_style = ...  # type: str
+__displayhook__: Any  # contains the original value of displayhook
+__excepthook__: Any  # contains the original value of excepthook
+exec_prefix: str
+executable: str
+float_repr_style: str
 hexversion: int
-last_type = ...  # type: Any
-last_value = ...  # type: Any
-last_traceback = ...  # type: Any
+last_type: Optional[Type[BaseException]]
+last_value: Optional[BaseException]
+last_traceback: Optional[TracebackType]
 maxsize: int
 maxunicode: int
-meta_path = ...  # type: List[Any]
-modules = ...  # type: Dict[str, Any]
-path = ...  # type: List[str]
-path_hooks = ...  # type: List[Any] # TODO precise type; function, path to finder
-path_importer_cache = ...  # type: Dict[str, Any] # TODO precise type
-platform = ...  # type: str
-prefix = ...  # type: str
-ps1 = ...  # type: str
-ps2 = ...  # type: str
-stdin = ...  # type: TextIO
-stdout = ...  # type: TextIO
-stderr = ...  # type: TextIO
-__stdin__ = ...  # type: TextIO
-__stdout__ = ...  # type: TextIO
-__stderr__ = ...  # type: TextIO
+meta_path: List[MetaPathFinder]
+modules: Dict[str, Any]
```

Comment:
Maybe `Dict[str, types.ModuleType]`?

Reply:
I thought about it but lazy loaders (and some other tools) can push into the dictionary just arbitrary object.
On another hand from static type checker perspective, the value is just a module (or compatible with module).
In a very rare case when mypy will fail on such tool's source code we can either ignore it or improve the stub.